CHANGELOG.md

```diff
@@ -5,6 +5,32 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## Release 10.0.0 (2020-06-03)
+
+[Changes since v10.0.0-mongodb.0](https://github.com/realm/realm-studio/compare/v10.0.0-mongodb.0...v10.0.0)
+
+### Breaking Changes
+
+- All functionality related to the Realm Object Server (ROS), such as connecting to a server, opening synchronized Realms from it and managing users and their permissions has been removed. ([#1282](https://github.com/realm/realm-studio/pull/1282))
+- All functionality related to the "legacy" Realm Cloud (available via https://cloud.realm.io/), such as authenticating and connecting to a server instance has been removed. ([#1282](https://github.com/realm/realm-studio/pull/1282))
+
+### Enhancements
+
+- Added support for creating & generating `ObjectId` when creating objects. ([#1291](https://github.com/realm/realm-studio/pull/1291))
+- Added support for editing existing `ObjectId` values. ([#1290](https://github.com/realm/realm-studio/pull/1290))
+- Primary keys in Class/Schema creation now defaults to an `ObjectId` property named `_id`.
+- Added support for creating `Decimal128` when creating objects. ([#1292](https://github.com/realm/realm-studio/pull/1292))
+- Added support for editing existing `Decimal128` values. ([#1292](https://github.com/realm/realm-studio/pull/1292))
+
+### Fixed
+
+- None
+
+### Internals
```

Comment:
We should probably add a "Compatibility section" similar to the SDK's stating which SDK's it can open files from. This is simple now as it's just v10 of all SDKs :-)

Reply:
I can technically open files from all previous SDKs? But ... it will upgrade the files :)

Reply:
Yeah, but that's only going to be true until the next major version :-)
SDK's changelogs also state that it will upgrade files.

Reply:
Perhaps we should list the file upgrade as a breaking change instead?

Reply:
It would be really helpful with a suggested change here.

Reply:
see https://github.com/realm/realm-studio/pull/1305 for suggestion